packages/dd-trace/src/telemetry/dependencies.js

```diff
@@ -31,11 +32,18 @@ function waitAndSend (config, application, host) {
 
 function onModuleLoad (data) {
   if (data) {
-    const { filename } = data
-    const request = data.request || getRequestFromFileName(filename)
+    let filename = data.filename
+    if (filename && filename.substring(0, 7) === FILE_PATH_START) {
```

Comment:
7 is a magic number, I think that it easier to understand `filename.substring(0, FILE_PATH_START.length)`